osu.Game.Tests/Visual/Editing/TestSceneHitObjectComposer.cs

```diff
@@ -161,10 +161,11 @@ public void TestDistanceSpacingHotkeys()
             AddStep("hold alt", () => InputManager.PressKey(Key.LAlt));
 
             AddStep("scroll mouse 5 steps", () => InputManager.ScrollVerticalBy(5));
-            AddAssert("distance spacing increased by 0.5", () => editorBeatmap.BeatmapInfo.DistanceSpacing == originalSpacing + 0.5);
 
             AddStep("release alt", () => InputManager.ReleaseKey(Key.LAlt));
             AddStep("release ctrl", () => InputManager.ReleaseKey(Key.LControl));
+
+            AddAssert("distance spacing increased by 0.5", () => editorBeatmap.BeatmapInfo.DistanceSpacing == originalSpacing + 0.5);
```

Comment:
I realise why this test was changed (it was because the holding down of <kbd>Alt</kbd> was toggling distance spacing off), but isn't this a bigger problem? Like, if you want to adjust spacing via <kbd>Ctrl</kbd>-<kbd>Alt</kbd>-mousewheel, then not seeing the changes you're making to the distance spacing while you're spinning the wheel seems like a pretty big usability issue, I'd say?

To illustrate visually:

https://user-images.githubusercontent.com/20418176/196784486-3f2a4f05-b8cc-451f-8c4e-446ef78c2595.mp4

Reply:
Guess we can make it always show when the ctrl+alt combo is used.

Reply:
One extra edge case is that currently toggle is handled per shift key (left / right) so you can toggle *twice* subsequently..  will look into a resolution for that as well.